### ISSUE TITLE
Error with computation requiring more parameters than supported 

### ISSUE THREAD

**Issue by AdamW1002** [pytorch/xla#3504]
## ❓ Questions and Help
Hi all, I am working in Colab with cloud tpu client and torch 1.11. I am using a dataset for code clone detection and using pre-trained CodeBERT for embeddings then building an FFN for binary classification on top of that.  My model outputs a single logit per input (with batch_size it actually comes out to a tensor of size [BATCH_SIZE]). I've tried batch sizes of 128 and 256 and both of them cause this error when I compute loss.  How can this be addressed?
![image](https://user-images.githubusercontent.com/72270600/163509907-898ddad8-1e15-4c35-abc0-02e4d70c39c3.png)
